### PR TITLE
Add 'Tomatoes' to fruits.txt

### PR DESCRIPTION
**Pull request by kr0stofa**
Literally title.
Some might argue that Tomatoes are not fruits, but I disagree. 
Don't @ me.

### CODE REVIEW

File: fruits.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 apples
 berries
 cherries
+Tomatoes
```
Comment (taneliang) [inline fruits.txt:4]: /nit tomatoes shouldn't be capitalized

**Review by taneliang — CHANGES_REQUESTED** [review]
Changes are of no value

Edit: This is true

**Comment by indocomsoft** [comment]
@Lessthanfree But is it to-MAY-to or to-MAH-to

File: fruits.txt (hunk 1 above)
Comment (kr0stofa) [reply fruits.txt:4]: I do what I want

**Pull request merged**